site/app/templates/GlobalHeader.twig

```diff
@@ -77,6 +77,7 @@
             </div>
             {% endif %}
             <div class="row" id="push">
+                <a id="skip-nav" href="#nav-positioner" tabindex="1">Skip to main content</a>
```

Comment:
Use of `tabindex` greater than zero is nearly always advised against in web accessibility standards.  q.v. https://webaim.org/techniques/keyboard/tabindex

Please remove the `tabindex` property and ensure tab order by logical placement of the link in HTML.

Reply:
I've made the changes